data.json

```diff
@@ -258,7 +258,7 @@
     "cantonese": "坟",
     "mandarin": "山坟",
     "english": "A simple grave",
-    "notes": "Literally 'a heap.'  The most simple type of graves: usually just an unmarked mud heap."
+    "notes": "Literally: “a heap.” The simplest type of graves: usually just an unmarked mud heap."
```

Comment:
This is a [left double quotation mark](https://unicode-table.com/en/201C/). We should probably be using a regular quotation mark, `"`, instead. It would need to be escaped in JSON as `\"`, since it is already within double quotations as a string.

Reply:
I'm not sure if we need to make the `"notes"` field consistent though. It is just freeform text.